sdk/keyvault/keyvault-certificates/test/utils/lro/restore/operation.ts

```diff
@@ -54,6 +54,7 @@ export interface RestoreCertificateBackupPollOperation
 /**
  * Reaches to the service and updates the restore certificate's poll operation.
  * @param options - The optional parameters, which are an abortSignal from \@azure/abort-controller and a function that triggers the poller's onProgress function.
+ * @this Mocha.Context
```

Comment:
"this" is not Mocha.Context here

Reply:
This might be a valid case where we should refactor the code, this has nothing to do with mocha